package.json

```diff
@@ -11,7 +11,11 @@
     "start": "cd frontend && npm start"
   },
   "dependencies": {
-    "react": "^16.6.3",
-    "react-router-dom": "^4.3.1"
+    "react": "^16.8.2",
+    "react-bootstrap-table-next": "^2.2.0",
+    "react-bootstrap-table2-paginator": "^2.0.3",
+    "react-dom": "^16.8.2",
+    "react-router-dom": "^4.3.1",
+    "reactstrap": "^7.1.0"
```

Comment:
I think we probably want to do npm install from the `frontend` directory, instead of the root directory. Otherwise we end up with packages scattered all over the place.